string behaviour
string img_path
string models_path